src/content/changes/highlights/2021-08-20.md

```diff
@@ -0,0 +1,19 @@
+---
+date: 2021-08-20T10:00:00
+title: Highlights for the week ending August 20, 2021
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Apps
+
+[Cert Manager v2.9.0](https://docs.giantswarm.io/changes/managed-apps/cert-manager-app/v2.9.0/) upgrades to upstream v1.4.2, deprecating `v1alpha2`, `v1alpha3`, and `v1beta1` versions of `cert-manager.io` and `acme.cert-manager.io` :warning: If you are upgrading the app on a cluster with Cert Manager upstream < `v1.0.0` ([Giant Swarm v2.2.5](https://github.com/giantswarm/cert-manager-app/blob/master/CHANGELOG.md#225---2020-09-29)), please make sure Cert Manager CRDs do not reference deprecated APIs. Read [upstream release notes](https://cert-manager.io/docs/release-notes/release-notes-1.4/).
```

Comment:
hi @ubergesundheit , can you check please for accuracy? 🙏

Reply:
Are you certain the `:warning:` renders to something meaningful in docs?